crates/driver/src/infra/persistence/mod.rs

```diff
@@ -0,0 +1,77 @@
+mod dto;
+
+use {
+    crate::{
+        domain::{
+            competition::{auction::Id, Auction},
+            liquidity,
+        },
+        infra::solver::Config,
+    },
+    serde::Deserialize,
+    std::sync::Arc,
+    tracing::Instrument,
+};
+
+#[derive(Clone, Debug, Default, Deserialize)]
+#[serde(rename_all = "kebab-case", deny_unknown_fields)]
+pub struct S3 {
+    /// The s3_instance_upload_* arguments configure how auction instances
+    /// should be uploaded to AWS S3.
+    /// They must either all be set or all not set.
+    pub s3_instance_upload_bucket: String,
+
+    /// Prepended to the auction id to form the final instance filename on S3.
+    /// Something like "staging/mainnet/"
+    pub s3_instance_upload_filename_prefix: String,
+}
```

Comment:
nit: How will this look like in the config? Maybe we can make it a subtype with shorter name a la:
```toml
[s3]
bucket = "..."
prefix = "..."
```